cpp/ps0001/ps0001_slon.cpp

```diff
@@ -2,15 +2,19 @@
 using namespace std;
 int main()
 {
-    int a;
-    int b;
-    int c;
+    // promptiung user for 3 numbers to add 1/2
+    short a;
+    short b;
+    short c;
     cin >> a; "\n";
     cin >> b; "\n";
     cin >> c; "\n";
+    
+    // outputting the result of the summation process 3/3
     cout <<"============================\n";
     cout <<"Answer is   ";
     cout << a + b + c;

```

Comment:
- [x] did summation process for the three inputted numbers
- [x] used `std::cout` in the right way to display output from the summation process
- [ ] output is as required
```
155
940
168
--------
1263
```
+1 bonus mark as the program is user-friendly